src/qt/bitcoingui.cpp

```diff
@@ -322,6 +324,15 @@ void BitcoinGUI::createActions()
     openAction = new QAction(platformStyle->TextColorIcon(":/icons/open"), tr("Open &URI..."), this);
     openAction->setStatusTip(tr("Open a bitcoin: URI or payment request"));
 
+    m_new_wallet_action = new QAction(platformStyle->TextColorIcon(":/icons/open"), tr("New &Wallet..."), this);
```

Comment:
IIUC the `&` sets the keyboard shortcut, so these should be unique (or absent).

Reply:
Then I think it's best to omit shortcuts for now, WDYT?

Reply:
I'm fine with either no shortcuts or unique ones.